pipeline/tasks/packman_tasks.py

```diff
@@ -23,7 +23,6 @@
 class PackmanTaskBase(task_base.TaskBase):
     def run_packman(self, language, api_name, *additional_args):
         # Fix the api_name convention (ex. logging-v2) for packman.
-        api_name = api_name.replace('-', '/')
```

Comment:
I moved it because it is needed to determine the behavior of `remove_api_dirs` in the gRPC-specific task.